modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1106,15 +1123,10 @@ protected function render_tab_titles_html( $item_settings ): string {
 
 		ob_start();
 		?>
-			<button <?php echo $render_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-				<span <?php echo $icon_class; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-					<?php echo $icon_html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-					<?php echo $icon_active_html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-				</span>
-				<span <?php echo $text_class; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-					<?php echo wp_kses_post( $title ); ?>
-				</span>
-			</button>
+		<span <?php $this->print_render_attribute_string( 'tab-icon' ); ?>>
+			<?php echo $icon_html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```

Comment:
let's add escaping here? wp_kses if it's just an html

Reply:
@mykytamurzin I have opened a separate task for the rendering of the SVG icons: https://elementor.atlassian.net/browse/ED-14938. 

Is it possible to approve this ticket, so that we won't delay the beta release.

I didn't change the rendering method or escaping method, but I agree it would be good if we can improve it.